openssl-sys/build.rs

```diff
@@ -20,10 +20,14 @@ fn main() {
         }
     }
 
-    let (libcrypto, libssl) = if target.contains("windows") {
-    	("eay32", "ssl32")
-    } else {
-    	("crypto", "ssl")
+    let libs_env = env::var("OPENSSL_LIBS").ok();
+    let libs = match libs_env {
+        Some(ref v) => v.split(":").collect(),
+        None => if target.contains("windows") && env::var_os("MSYSTEM").is_none() {
```

Comment:
Let's leave the MSYSTEM check out for now. We may be moving to openssl being provided by a separate crate on Windows to avoid the linkage nightmare that it is right now.